lnwallet/test/test_interface.go

```diff
@@ -1311,13 +1311,64 @@ func testListTransactionDetails(miner *rpctest.Harness,
 
 	// Query for transactions only in the latest block. We do not expect
 	// any transactions to be returned.
-	txDetails, err = alice.ListTransactionDetails(
+	txDetails, _, _, err = alice.ListTransactionDetails(
 		chainTip, chainTip, "", 0, 1000,
 	)
 	require.NoError(t, err, "unexpected error")
 	if len(txDetails) != 0 {
 		t.Fatalf("expected 0 transactions, got: %v", len(txDetails))
 	}
+
+	// Next we try to query for only 5 transactions.
+	var (
+		firstIdx, lastIdx uint64
+	)
+	txDetails, firstIdx, lastIdx, err = alice.ListTransactionDetails(
+		startHeight, chainTip, "", 0, 5,
+	)
+	require.NoError(t, err, "unexpected error")
```

Comment:
nit: the "unexpected error" string isn't really required. The testing library will already format it in a way that it's clear the error is not expected.